Mathlib/Topology/Constructions.lean

```diff
@@ -1038,6 +1038,10 @@ theorem isOpenMap_sum {f : X ⊕ Y → Z} :
   simp only [isOpenMap_iff_nhds_le, Sum.forall, nhds_inl, nhds_inr, Filter.map_map, comp]
 #align is_open_map_sum isOpenMap_sum
 
+theorem IsOpenMap.sum_map {f : X → Y} {g : Z → W} (hf : IsOpenMap f) (hg : IsOpenMap g) :
```

Comment:
Following the [naming convention](https://leanprover-community.github.io/contribute/naming.html), the foreign namespaces should be appended back in camel-case unless the unnamespaced name is unambiguous.
```suggestion
theorem IsOpenMap.sumMap {f : X → Y} {g : Z → W} (hf : IsOpenMap f) (hg : IsOpenMap g) :
```